src/prefect/cli/_prompts.py

```diff
@@ -328,21 +328,27 @@ def prompt_schedule_type(console):
     return selection["type"]
 
 
-def prompt_schedule(console) -> SCHEDULE_TYPES:
+def prompt_schedule(console) -> Tuple[SCHEDULE_TYPES, bool]:
     """
     Prompt the user for a schedule type. Once a schedule type is selected, prompt
     the user for the schedule details and return the schedule.
     """
     schedule_type = prompt_schedule_type(console)
     if schedule_type == "Cron":
-        return prompt_cron_schedule(console)
+        schedule = prompt_cron_schedule(console)
     elif schedule_type == "Interval":
-        return prompt_interval_schedule(console)
+        schedule = prompt_interval_schedule(console)
     elif schedule_type == "RRule":
-        return prompt_rrule_schedule(console)
+        schedule = prompt_rrule_schedule(console)
     else:
         raise Exception("Invalid schedule type")
 
+    is_schedule_active = confirm(
+        "Would you like to activate this schedule?", default=True
+    )
```

Comment:
i'm not sure if it makes more sense to say 'activate this schedule' (which to me sounds a little confusing because  the schedule is always listed on the deployment regardless).. rather than something about the deployment being paused or not.

Reply:
Right, my thinking here was that the next part of of the enhanced-scheduling raft is to make deployment->schedules a one->many relation, in which case this wouldn't be talking about the _deployment_ being paused it would in fact be about the schedule.